src/components/pages/Alerts/RenderAlerts.js

```diff
@@ -9,52 +9,19 @@ import { AdminContext } from '../../../state/contexts';
 import { getClubs, getMembersReaction } from '../../../state/actions';
 import { LoadingComponent } from '../../common';
 import '../../../styles/styles.css';
+import axios from 'axios';
 
 const { Content, Sider } = Layout;
 
-function RenderAlerts(props) {
+function RenderAlerts() {
   const context = useContext(AdminContext);
-  // const [isModalVisible, setIsModalVisible] = useState(false);
-
-  // const handleOk = () => {
-  //   // Send data to backend
-  //   // if (inputData.individual.length > 0) {
-  //   //   inputData.individual.forEach(item => {
-  //   //     console.log(item);
-  //   //     postClub(item);
-  //   //   });
-  //   // }
-  //   // setIsModalVisible(false);
-  //   // clearState();
-  //   // setTimeout(() => {
-  //   //   fetchClubs();
-  //   // }, 2000);
-  // };
-
-  // const handleCancel = () => {
-  //   setIsModalVisible(false);
-  //   clearState();
-  // };
-
-  // const clearState = () => {
-  //   setInputData({
-  //     individual: [],
-  //     file: [],
-  //   });
-  // };
-
-  // const showModal = () => {
-  //   setIsModalVisible(true);
-  // };
+  const [resolve, setResolve] = useState();
 
   useEffect(() => {
-    if (context.clubs.length === 0) {
-      getClubs('authState', context);
-    }
-    if (context.memberReactions.length === 0) {
-      getMembersReaction('authState', context);
-    }
-  }, []);
+    setResolve(true);
```

Comment:
Why do we have setResolve(true) here, but not when declaring state as initial state?

Reply:
When I set the initial state to useState(boolean) the resolve button didn't re-render until the third click I'm not entirely sure why but on the first click it would not cause any change, except for the tally of the alerts would go down by 1 however the component wouldn't re-render, then if I click it again the tally goes up instead. then for a reason I don't really understand after the third click it would trigger a re-render with the correct tally after removing that alert.  Since I wanted the change in state to trigger a re-render, I chose to just set the state at the triggering of the useEffect.